src/main/drivers/flash_m25p16.c

```diff
@@ -57,6 +57,7 @@
 #define JEDEC_ID_WINBOND_W25Q64        0xEF4017
 #define JEDEC_ID_WINBOND_W25Q128       0xEF4018
 #define JEDEC_ID_WINBOND_W25Q256       0xEF4019
+#define JEDEC_ID_CYPRESS_S25FL128L     0x016018
```

Comment:
Can we please have the change to the flash driver in a separate pull request, so that firmware changes and target updates can be kept separate?

Reply:
No problem, I close this and create 2 different PR.